tools/packaging/kernel/build-kernel.sh

```diff
@@ -147,7 +147,7 @@ get_kernel() {
 	[ -n "${kernel_path}" ] || die "kernel_path not provided"
 	[ ! -d "${kernel_path}" ] || die "kernel_path already exist"
 
-	if [ "${conf_guest}" != "" ]; then
+	if [ "${conf_guest}" == "tdx" ]; then
```

Comment:
Hmm.  No, please, take a look at: https://github.com/kata-containers/kata-containers/pull/4861#discussion_r942103605

Also https://github.com/kata-containers/kata-containers/blob/1b92a946d6c416b8fd1b2cb201bbe4c8669531c9/versions.yaml#L162-L165, so we can and should be using this as it's.